include/bitcoin/explorer/commands/qrcode.hpp

```diff
@@ -125,7 +127,8 @@ class BCX_API qrcode
     virtual arguments_metadata& load_arguments()
     {
         return get_argument_metadata()
-            .add("PAYMENT_ADDRESS", 1);
+            .add("PAYMENT_ADDRESS", 1)
+            .add("filename", 1);
```

Comment:
Hrm, so you'd prefer that the binary data just be output to stdout on success and not have a file write option?  I had based the usage on the commented out bash script that used qrencode and wrote to a file argument.